Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 565)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -23,6 +23,12 @@
   install-data          \
   install-load          \
   install-schema        \
+  uninstall             \
+  uninstall-after       \
+  uninstall-before      \
+  uninstall-data        \
+  uninstall-load        \
+  uninstall-schema      \
 
 
 
@@ -300,14 +306,10 @@
 # При первоначальной установке модуля в БД должно указываться значение "Last".
 INSTALL_VERSION  =
 
-# Стандартные действия, выполняемые перед установкой.
-# Действия выполняются скриптами, входящими в OMS.
-# Значение переменной не должно изменяться в модуле ( в случае необходимости
-# можно убирать использование этой переменной из installBeforeTarget и указывать
-# там вручную необходимые стандартные действия).
-installBeforeOmsAction =                      \
-  oms-check-lock.sql                          \
-  oms-show-invalid.sql                        \
+# Список объектов для обязательной проверки блокировок перед установкой и
+# отменой установки.
+# Формат списка такой же, как у loadTarget.
+mandatoryCheckLockTarget =
 
 
 
@@ -318,9 +320,14 @@
 checkLockTarget =                             \
   $(loadTarget)                               \
 
-# Список объектов для обязательной проверки блокировок перед установкой.
-# Формат списка такой же, как у checkLockTarget.
-mandatoryCheckLockTarget =
+# Стандартные действия, выполняемые перед установкой.
+# Действия выполняются скриптами, входящими в OMS.
+# Значение переменной не должно изменяться в модуле ( в случае необходимости
+# можно убирать использование этой переменной из installBeforeTarget и указывать
+# там вручную необходимые стандартные действия).
+installBeforeOmsAction =                      \
+  oms-check-lock.sql                          \
+  oms-show-invalid.sql                        \
 
 
 
@@ -490,6 +497,161 @@
 
 
 #
+# group: Отмена установки модуля в БД
+#
+
+# Список объектов для проверки отсутствия блокировок перед отменой установки.
+# Из списка автоматически исключаются файлы, не загружаемые из-за значения
+# параметра SKIP_FILE_MASK.
+# Формат списка такой же, как у loadTarget.
+uninstallCheckLockTarget = \
+  $(uninstallLoadTarget) \
+
+
+# Стандартные действия, выполняемые перед отменой установки.
+# Действия выполняются скриптами, входящими в OMS.
+# Значение переменной не должно изменяться в модуле ( в случае необходимости
+# можно убирать использование этой переменной из uninstallBeforeTarget и
+# указывать там вручную необходимые стандартные действия).
+uninstallBeforeOmsAction = \
+  oms-check-lock.sql \
+  oms-show-invalid.sql \
+
+
+
+# Действия, выполняемые перед отменой установки модуля.
+uninstallBeforeTarget = \
+  $(addsuffix .$(ru), \
+    $(if $(INSTALL_VERSION), \
+      $(wildcard Install/Config/$(INSTALL_VERSION)/Revert/before-action.sql),) \
+    $(wildcard Install/Config/before-action.sql) \
+    $(uninstallBeforeOmsAction) \
+  ) \
+  $(addsuffix .$(ru2), \
+    $(uninstallBeforeOmsAction) \
+  ) \
+  $(addsuffix .$(ru3), \
+    $(uninstallBeforeOmsAction) \
+  ) \
+
+
+
+# target: uninstall-before
+# Выполняет предварительные действия перед отменой установки.
+
+uninstall-before: uninstall-before.oms
+
+
+
+# Скрипты для отмены установки схемы
+uninstallSchemaTarget = \
+  $(addsuffix .$(ru), \
+    $(wildcard Install/Schema/$(INSTALL_VERSION)/revert.sql) \
+  ) \
+  $(addsuffix .$(ru2), \
+  ) \
+  $(addsuffix .$(ru3), \
+  ) \
+
+
+
+# target: uninstall-schema
+# Отменяет изменения, внесенные в объекты схемы при выполнении установки
+# ( по цели <install-schema>).
+# Для отмены изменений, внесенных в объекты схемы, выполняются скрипты из
+# каталога Install/Schema/$(INSTALL_VERSION).
+
+uninstall-schema: uninstall-schema.oms
+
+
+
+# Файлы с предыдущими версиями объектов, загружаемыми при отмене установки.
+uninstallLoadTarget = \
+  $(addsuffix .$(ru), \
+    $(if $(INSTALL_VERSION), \
+      $(foreach m, *.typ *.vw *.pks *.trg *.tyb *.pkb, \
+        $(wildcard Install/Schema/$(INSTALL_VERSION)/Revert/$(m)) \
+      ) \
+    ,) \
+  ) \
+  $(addsuffix .$(ru2), \
+  ) \
+  $(addsuffix .$(ru3), \
+  ) \
+
+
+
+# target: uninstall-load
+# Загружает предыдущие версии объектов в БД при отмене установки.
+
+uninstall-load: uninstall-load.oms
+
+
+
+# Скрипты для отмены изменений, внесенных при загрузке установочных данных в БД.
+uninstallDataTarget = \
+  $(addsuffix .$(ru), \
+    $(wildcard Install/Data/$(INSTALL_VERSION)/revert.sql) \
+  ) \
+  $(addsuffix .$(ru2), \
+  ) \
+  $(addsuffix .$(ru3), \
+  ) \
+
+
+
+# target: uninstall-data
+# Отменяет изменения, внесенные при загрузке установочных данные в БД
+# ( по цели <install-data>).
+
+uninstall-data: uninstall-data.oms
+
+
+
+# Стандартные действия, выполняемые после отмены установки.
+# Действия выполняются скриптами, входящими в OMS.
+# Значение переменной не должно изменяться в модуле ( в случае необходимости
+# можно убирать использование этой переменной из uninstallAfterTarget и
+# указывать там вручную необходимые стандартные действия).
+uninstallAfterOmsAction = \
+  oms-compile-invalid.sql \
+
+
+
+# Действия, выполняемые после отмены установки.
+uninstallAfterTarget = \
+  $(addsuffix .$(ru), \
+    $(uninstallAfterOmsAction) \
+    $(wildcard Install/Config/after-action.sql) \
+    $(if $(INSTALL_VERSION), \
+      $(wildcard Install/Config/$(INSTALL_VERSION)/Revert/after-action.sql),) \
+  ) \
+  $(addsuffix .$(ru2), \
+    $(uninstallAfterOmsAction) \
+  ) \
+  $(addsuffix .$(ru3), \
+    $(uninstallAfterOmsAction) \
+  ) \
+
+
+
+# target: uninstall-after
+# Выполняет завершающие отмену установки действия.
+
+uninstall-after: uninstall-after.oms
+
+
+
+# target: uninstall
+# Отменяет установку версии модуля в БД.
+# Версия, установка которой отменяется, определяется значением параметра
+# <INSTALL_VERSION>.
+
+uninstall: uninstall.oms
+
+
+
+#
 # group: Выдача прав пользователям БД
 #
 
