Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 633)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -56,6 +56,36 @@

 # target: show-oms-version
 # Показывает версию OMS-файлов, входящих в состав модуля.
+#
+# Если же нужно проверить актуальность установленной версии
+# системы OMS модулю, что чаще необходимо перед установкой модуля,
+# можно выполнить команду:
+# (code)
+#    oms-load --version
+# (end code)
+#
+# В случае, если версия ( правка, т.е. ревизия) OMS младше,
+# чем версия OMS-файлов модуля, рекомендуется обновить ( установить) OMS.
+#
+# Пример:
+# (code)
+#
+# $ make show-oms-version
+# make[1]: Entering directory `/cygdrive/c/svn/DWH/Module/CredilogicMView/Trunk/DB'
+# OMS files version: 1.5.2 ( rev. 638, 2010-08-12)
+# make[1]: Leaving directory `/cygdrive/c/svn/DWH/Module/CredilogicMView/Trunk/DB'
+#
+# $ oms-load --version
+# oms-load (OMS) 1.5.2
+# SVN Version Information:
+#  Module root     : Oracle/Module/OraMakeSystem
+#  File revision   : 648
+#  File change date: 2010-08-13 15:32:51 +0400
+# (end code)
+#
+# OK. Версия OMS старше, чем версия файлов модуля CreidlogicMView.

 show-oms-version: show-oms-version.oms

@@ -95,6 +125,13 @@



+# target: gendoc-menu
+# Генерирует меню и документацию.
+
+gendoc-menu: gendoc-menu.oms
+
+
+
 #
 # group: Загрузка файлов в БД
 #
