Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 419)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -175,6 +175,17 @@
 # ( если есть).
 LOAD_OPERATORID := $(ORACLE_OPERATOR)
 
+# build var: SQL_DEFINE
+# Список значений для переменных SQL*Plus.
+#
+# Переменные создаются командой define перед загрузкой каждого файла.
+# Список указывается в формате name=value[,name2=value2]..., для экранирования
+# разделителя может использоваться обратный слэш, при наличии пробелов нужно
+# заключать список в кавычки, пробелы в части имени игнорируются ( например,
+# SQL_DEFINE="name1=a \, b, name2 =c" задает переменную name1 со значением
+# "a , b"  и переменную name2 со значением "c").
+SQL_DEFINE :=
+
 # build var: SKIP_FILE_MASK
 # Маски для исключения выполнения ( загрузки) файлов в БД.
 #
