Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 2128)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -644,14 +644,14 @@
 
 # Скрипты для загрузки пакетных заданий и их параметров в БД.
 installBatchTarget = \
-  $(addsuffix .$(ru), \
+  $(addsuffix .$(lu), \
     $(wildcard Install/Batch/Last/*.job.sql) \
     $(wildcard Install/Batch/Last/*/*.job.sql) \
     $(wildcard Install/Batch/Last/*/batch.xml) \
   ) \
-  $(addsuffix .$(ru2), \
+  $(addsuffix .$(lu2), \
   ) \
-  $(addsuffix .$(ru3), \
+  $(addsuffix .$(lu3), \
   ) \
 
 
