Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 400)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -231,6 +231,7 @@
 # ƒействи€, выполн€емые перед установкой модул€.
 installBeforeTarget =													\
 	$(addsuffix .$(ru),													\
+		$(wildcard Install/Action/$(INSTALL_VERSION)/before.sql)	\
 		$(wildcard Install/Action/before.sql)			\
 		$(installBeforeOmsAction)									\
 	)																						\
@@ -285,6 +286,7 @@
 	$(addsuffix .$(ru),													\
 		$(installAfterOmsAction)									\
 		$(wildcard Install/Action/after.sql)			\
+		$(wildcard Install/Action/$(INSTALL_VERSION)/after.sql)	\
 	)																						\
 	$(addsuffix .$(ru2),												\
 		$(installAfterOmsAction)									\
