Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 1643)
+++ Config/NewModule/DB/Makefile	(revision 1644)
@@ -23,6 +23,7 @@
   load-clean            \
   install               \
   install-after         \
+  install-batch         \
   install-before        \
   install-data          \
   install-load          \
@@ -591,6 +592,27 @@
 
 
 
+# Скрипты для загрузки пакетных заданий и их параметров в БД.
+installBatchTarget = \
+  $(addsuffix .$(ru), \
+	  $(wildcard Install/Batch/Last/*.job.sql) \
+	  $(wildcard Install/Batch/Last/*/*.job.sql) \
+	  $(wildcard Install/Batch/Last/*/batch.xml) \
+  )
+  $(addsuffix .$(ru2), \
+  ) \
+  $(addsuffix .$(ru3), \
+  ) \
+
+
+
+# target: install-batch
+# Устанавливает пакетные задания в БД.
+
+install-batch: install-batch.oms
+
+
+
 # Стандартные действия, выполняемые после установки.
 # Действия выполняются скриптами, входящими в OMS.
 # Значение переменной не должно изменяться в модуле ( в случае необходимости
