Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 413)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -4,22 +4,22 @@
 omsModuleDir = OmsModule
 
 # Абстрактные цели
-.PHONY:									\
-	all 									\
-	clean									\
-	show-oms-version			\
-	update-oms-version		\
-	gendoc								\
-	gendoc-clean					\
-	load									\
-	load-clean						\
-	install								\
-	install-after					\
-	install-before				\
-	install-schema				\
-	
+.PHONY:                  \
+  all                   \
+  clean                  \
+  show-oms-version      \
+  update-oms-version    \
+  gendoc                \
+  gendoc-clean          \
+  load                  \
+  load-clean            \
+  install                \
+  install-after          \
+  install-before        \
+  install-schema        \
 
 
+
 #
 # group: Общие цели
 #
@@ -58,15 +58,15 @@
 
 #
 # group: Генерация документации
-# 
+#
 
 # build var: GENDOC_DB_DIR
 # Каталог для генерации документации по DB-части проекта.
-GENDOC_DB_DIR			= ../Doc/DB/AutoGen
+GENDOC_DB_DIR      = ../Doc/DB/AutoGen
 
 # build var: GENDOC_DB_FLAGS
 # Дополнительные параметры генерации документации.
-GENDOC_DB_FLAGS		=
+GENDOC_DB_FLAGS    =
 
 
 # target: gendoc
@@ -90,85 +90,85 @@
 # Функции для разбора строки подключения.
 
 # Выделяет имя пользователя из loadUserId.
-getUserName	= $(firstword $(subst /, ,$(firstword $(subst @, ,$(loadUserId)))))
+getUserName  = $(firstword $(subst /, ,$(firstword $(subst @, ,$(loadUserId)))))
 
 # Выделяет имя БД из loadUserId
-getDbName		= \
-	$(word \
-		$(words $(subst @, ,$(loadUserId)x)) \
-		, $(subst @, ,$(loadUserId)) \
-	)
-	
+getDbName    = \
+  $(word \
+    $(words $(subst @, ,$(loadUserId)x)) \
+    , $(subst @, ,$(loadUserId)) \
+  )
+
 # Выделяет пользователя ( без пароля) из loadUserId.
-getUser			= $(getUserName)@$(getDbName)
+getUser      = $(getUserName)@$(getDbName)
 
 
 # build var: LOAD_DB
 # БД для загрузки.
-LOAD_DB			= 
+LOAD_DB      =
 
 # build var: LOAD_USERID
 # Пользователь для загрузки ( формат: [userName[/password]][@dbName] ).
-# 
+#
 # Имеет приоритет в случае противоречия с <LOAD_DB>.
-LOAD_USERID = 
+LOAD_USERID =
 
 # Устанавливаем LOAD_DB, если имя БД заданo в LOAD_USERID
 ifneq ($(LOAD_USERID),)
-loadUserId					:= $(LOAD_USERID)
-loadDb							:= $(getDbName)
+loadUserId          := $(LOAD_USERID)
+loadDb              := $(getDbName)
 ifneq ($(loadDb),)
 ifneq ($(loadDb),$(loadUserId))
-override LOAD_DB		:= $(loadDb)
+override LOAD_DB    := $(loadDb)
 endif
 endif
 endif
 
 # build var: LOAD_DB2
 # Вторая БД для загрузки.
-LOAD_DB2		= 
+LOAD_DB2    =
 
 # build var: LOAD_USERID2
 # Второй пользователь для загрузки ( формат: [userName[/password]][@dbName] ).
 #
 # Имеет приоритет в случае противоречия с <LOAD_DB2>.
-LOAD_USERID2 = 
+LOAD_USERID2 =
 
 # Устанавливаем LOAD_DB2, если имя БД заданo в LOAD_USERID2
 ifneq ($(LOAD_USERID2),)
-loadUserId					:= $(LOAD_USERID2)
-loadDb							:= $(getDbName)
+loadUserId          := $(LOAD_USERID2)
+loadDb              := $(getDbName)
 ifneq ($(loadDb),)
 ifneq ($(loadDb),$(loadUserId))
-override LOAD_DB2		:= $(loadDb)
+override LOAD_DB2    := $(loadDb)
 endif
 endif
 endif
 
 # build var: LOAD_DB3
 # Третья БД для загрузки.
-LOAD_DB3		=
+LOAD_DB3    =
 
 # build var: LOAD_USERID3
 # Третий пользователь для загрузки ( формат: [userName[/password]][@dbName] ).
 #
 # Имеет приоритет в случае противоречия с <LOAD_DB3>.
-LOAD_USERID3 = 
+LOAD_USERID3 =
 
 # Устанавливаем LOAD_DB3, если имя БД заданo в LOAD_USERID3
 ifneq ($(LOAD_USERID3),)
-loadUserId					:= $(LOAD_USERID3)
-loadDb							:= $(getDbName)
+loadUserId          := $(LOAD_USERID3)
+loadDb              := $(getDbName)
 ifneq ($(loadDb),)
 ifneq ($(loadDb),$(loadUserId))
-override LOAD_DB3		:= $(loadDb)
+override LOAD_DB3    := $(loadDb)
 endif
 endif
 endif
 
 # build var: LOAD_OPERATORID
 # Оператор для загрузки ( формат: operatorName[/password] ).
-# 
+#
 # В случае наличия значения, сразу после подключения производится
 # регистрация оператора ( через модуль Operator) в БД.
 # По умолчанию используется значение переменной окружения ORACLE_OPERATOR
@@ -177,7 +177,7 @@
 
 # build var: SKIP_FILE_MASK
 # Маски для исключения выполнения ( загрузки) файлов в БД.
-# 
+#
 # Представляет собой список ( через запятую) масок на путь к файлам
 # относительно каталога DB ( можно использовать спецсимволы "*" и "?").
 # Файлы, соответствующие маске, не выполняются ( не загружаются) в БД.
@@ -185,7 +185,7 @@
 
 # build var: LOAD_FILE_MASK
 # Маски для ограничения загружаемых по цели load файлов.
-# 
+#
 # Представляет собой список ( через запятую) масок на путь к загружаемым файлам
 # относительно каталога DB ( можно использовать спецсимволы "*" и "?").
 # Если маска задана, то файлы, которые должны загружаться по цели load и не
@@ -193,14 +193,14 @@
 LOAD_FILE_MASK :=
 
 # Файлы для загрузки
-loadTarget = 														\
-	$(addsuffix .$(lu),										\
-		$(wildcard *.pk[s,b])								\
-	)																			\
-	$(addsuffix .$(lu2),									\
-	)																			\
-	$(addsuffix .$(lu3),									\
-	)																			\
+loadTarget =                             \
+  $(addsuffix .$(lu),                    \
+    $(wildcard *.pk[s,b])                \
+  )                                      \
+  $(addsuffix .$(lu2),                  \
+  )                                      \
+  $(addsuffix .$(lu3),                  \
+  )                                      \
 
 
 
@@ -225,31 +225,31 @@
 # build var: INSTALL_VERSION
 # Версия модуля для устновки в БД.
 # При первоначальной установке модуля в БД должно указываться значение "Last".
-INSTALL_VERSION	= 
+INSTALL_VERSION  =
 
 # Стандартные действия, выполняемые перед установкой.
 # Действия выполняются скриптами, входящими в OMS.
 # Значение переменной не должно изменяться в модуле ( в случае необходимости
 # можно убирать использование этой переменной из installBeforeTarget и указывать
 # там вручную необходимые стандартные действия).
-installBeforeOmsAction =											\
-	oms-show-invalid.sql												\
+installBeforeOmsAction =                      \
+  oms-show-invalid.sql                        \
 
 
 
 # Действия, выполняемые перед установкой модуля.
-installBeforeTarget =													\
-	$(addsuffix .$(ru),													\
-		$(wildcard Install/Config/$(INSTALL_VERSION)/before-action.sql)	\
-		$(wildcard Install/Config/before-action.sql)			\
-		$(installBeforeOmsAction)									\
-	)																						\
-	$(addsuffix .$(ru2),												\
-		$(installBeforeOmsAction)									\
-	)																						\
-	$(addsuffix .$(ru3),												\
-		$(installBeforeOmsAction)									\
-	)																						\
+installBeforeTarget =                          \
+  $(addsuffix .$(ru),                          \
+    $(wildcard Install/Config/$(INSTALL_VERSION)/before-action.sql)  \
+    $(wildcard Install/Config/before-action.sql)      \
+    $(installBeforeOmsAction)                  \
+  )                                            \
+  $(addsuffix .$(ru2),                        \
+    $(installBeforeOmsAction)                  \
+  )                                            \
+  $(addsuffix .$(ru3),                        \
+    $(installBeforeOmsAction)                  \
+  )                                            \
 
 
 
@@ -261,14 +261,14 @@
 
 
 # Скрипты для выполнения установки схемы
-installSchemaTarget = 									\
-	$(addsuffix .$(lu),										\
-		$(wildcard Install/Schema/$(INSTALL_VERSION)/run.sql)			\
-	)																			\
-	$(addsuffix .$(lu2),									\
-	)																			\
-	$(addsuffix .$(lu3),									\
-	)																			\
+installSchemaTarget =                   \
+  $(addsuffix .$(lu),                    \
+    $(wildcard Install/Schema/$(INSTALL_VERSION)/run.sql)      \
+  )                                      \
+  $(addsuffix .$(lu2),                  \
+  )                                      \
+  $(addsuffix .$(lu3),                  \
+  )                                      \
 
 
 
@@ -286,24 +286,24 @@
 # Значение переменной не должно изменяться в модуле ( в случае необходимости
 # можно убирать использование этой переменной из installAfterTarget и указывать
 # там вручную необходимые стандартные действия).
-installAfterOmsAction =												\
-	oms-compile-invalid.sql											\
+installAfterOmsAction =                        \
+  oms-compile-invalid.sql                      \
 
 
 
 # Действия, выполняемые после установки.
-installAfterTarget =													\
-	$(addsuffix .$(ru),													\
-		$(installAfterOmsAction)									\
-		$(wildcard Install/Config/after-action.sql)			\
-		$(wildcard Install/Config/$(INSTALL_VERSION)/after-action.sql)	\
-	)																						\
-	$(addsuffix .$(ru2),												\
-		$(installAfterOmsAction)									\
-	)																						\
-	$(addsuffix .$(ru3),												\
-		$(installAfterOmsAction)									\
-	)																						\
+installAfterTarget =                          \
+  $(addsuffix .$(ru),                          \
+    $(installAfterOmsAction)                  \
+    $(wildcard Install/Config/after-action.sql)      \
+    $(wildcard Install/Config/$(INSTALL_VERSION)/after-action.sql)  \
+  )                                            \
+  $(addsuffix .$(ru2),                        \
+    $(installAfterOmsAction)                  \
+  )                                            \
+  $(addsuffix .$(ru3),                        \
+    $(installAfterOmsAction)                  \
+  )                                            \
 
 
 
