Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 1328)
+++ Config/NewModule/DB/Makefile	(revision 1329)
@@ -186,6 +186,14 @@
 # Настройки локали, используемые при загрузке файлов в БД.
 export NLS_LANG = AMERICAN_CIS.CL8MSWIN1251
 
+# build var: PLSQL_WARNINGS
+# Уровни предупреждения компилятора PL/SQL.
+# Соответствуют параметру plsql_warnings сеанса Oracle.
+#
+# Пример значения: "ENABLE:SEVERE"
+#
+PLSQL_WARNINGS =
+
 # build var: LOAD_DB
 # БД для загрузки.
 LOAD_DB =
