Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 1095)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -423,7 +423,7 @@
 #
 
 # build var: INSTALL_VERSION
-# Версия модуля для устновки в БД.
+# Версия модуля для установки в БД.
 # При первоначальной установке модуля в БД должно указываться значение "Last".
 INSTALL_VERSION  =
 
