Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 602)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -240,7 +240,7 @@
 # заключать список в кавычки, пробелы в части имени игнорируются ( например,
 # SQL_DEFINE="name1=a \, b, name2 =c" задает переменную name1 со значением
 # "a , b"  и переменную name2 со значением "c").
-SQL_DEFINE :=
+SQL_DEFINE =
 
 # build var: SKIP_FILE_MASK
 # Маски для исключения выполнения ( загрузки) файлов в БД.
