Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 1174)
+++ Config/NewModule/DB/Makefile	(revision 1175)
@@ -617,6 +617,28 @@



+# Скрипты для загрузки в БД объектов для тестирования
+installTestTarget = \
+  $(addsuffix .$(lu), \
+    $(wildcard \
+      $(addprefix Test/, \
+        *.typ *.pks *.tyb *.pkb \
+      ) \
+    ) \
+  ) \
+  $(addsuffix .$(lu2), \
+  ) \
+  $(addsuffix .$(lu3), \
+  ) \
+
+
+
+# target: install-test
+# Загружает объекты для тестирования в БД.
+install-test: install-test.oms
+
+
+
 # target: install-save-info
 # Сохраняет в БД информацию об установке модуля.

