Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 274)
+++ Config/NewModule/DB/Makefile	(revision 275)
@@ -71,7 +71,7 @@
 # Выделяет имя БД из loadUserId
 getDbName		= \
 	$(word \
-		$(words $(subst @, ,$(loadUserId))) \
+		$(words $(subst @, ,$(loadUserId)x)) \
 		, $(subst @, ,$(loadUserId)) \
 	)
 	
@@ -94,9 +94,11 @@
 loadUserId					:= $(LOAD_USERID)
 loadDb							:= $(getDbName)
 ifneq ($(loadDb),)
+ifneq ($(loadDb),$(loadUserId))
 override LOAD_DB		:= $(loadDb)
 endif
 endif
+endif
 
 # build var: LOAD_DB2
 # Вторая БД для загрузки.
@@ -113,9 +115,11 @@
 loadUserId					:= $(LOAD_USERID2)
 loadDb							:= $(getDbName)
 ifneq ($(loadDb),)
+ifneq ($(loadDb),$(loadUserId))
 override LOAD_DB2		:= $(loadDb)
 endif
 endif
+endif
 
 # build var: LOAD_DB3
 # Третья БД для загрузки.
@@ -132,9 +136,11 @@
 loadUserId					:= $(LOAD_USERID3)
 loadDb							:= $(getDbName)
 ifneq ($(loadDb),)
+ifneq ($(loadDb),$(loadUserId))
 override LOAD_DB3		:= $(loadDb)
 endif
 endif
+endif
 
 # build var: LOAD_OPERATORID
 # Оператор для загрузки ( формат: operatorName[/password] ).
