Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 394)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -214,11 +214,10 @@
 # group: Установка модуля в БД
 #
 
-# build var: SCHEMA_VERSION
-# Версия схемы для устновки в БД.
-# Для выполнения установки выполняются скрипты из каталога
-# Install/Schema/$(SCHEMA_VERSION).
-SCHEMA_VERSION	= 
+# build var: INSTALL_VERSION
+# Версия модуля для устновки в БД.
+# При первоначальной установке модуля в БД должно указываться значение "Last".
+INSTALL_VERSION	= 
 
 # Стандартные действия, выполняемые перед установкой.
 # Действия выполняются скриптами, входящими в OMS.
@@ -254,7 +253,7 @@
 # Скрипты для выполнения установки схемы
 installSchemaTarget = 									\
 	$(addsuffix .$(lu),										\
-		$(wildcard Install/Schema/$(SCHEMA_VERSION)/run.sql)			\
+		$(wildcard Install/Schema/$(INSTALL_VERSION)/run.sql)			\
 	)																			\
 	$(addsuffix .$(lu2),									\
 	)																			\
@@ -265,8 +264,8 @@
 
 # target: install-schema
 # Устанавливает объекты схемы в БД.
-# Должна быть указана версия схемы ( в <SCHEMA_VERSION>) и параметры подключения
-# к БД ( в LOAD_USERID*).
+# Для выполнения установки/обновления объектов схемы выполняются скрипты из
+# каталога Install/Schema/$(INSTALL_VERSION).
 
 install-schema: install-schema.oms
 
