Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 967)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -310,6 +310,32 @@
 # ( если есть).
 LOAD_OPERATORID := $(ORACLE_OPERATOR)
 
+
+
+# build var: LOCAL_DB_DIR
+# Имя подкаталога для загрузки локальных скриптов, относящихся к определенной
+# БД.
+# Определяется автоматически на основе значения параметра <LOAD_DB9>
+# ( <LOAD_USERID9>).
+LOCAL_DB_DIR = $(call getProductionDbName,$(LOAD_DB9))
+
+# Обеспечиваем корректное значение переменной
+override LOCAL_DB_DIR := $(call getLocalDbDir,$(LOCAL_DB_DIR))
+
+# build var: LOCAL_USER_DIR
+# Расположение подкаталога для загрузки локальных скриптов, относящихся к
+# определенному пользователю БД ( формат: dbDirName/userDirName).
+# Определяется автоматически на основе значения параметров <LOCAL_DB_DIR> и
+# <LOAD_USERID9>.
+#
+LOCAL_USER_DIR = \
+  $(addsuffix /$(call getUserName,$(LOAD_USERID9)),$(LOCAL_DB_DIR))
+
+# Обеспечиваем корректное значение переменной
+override LOCAL_USER_DIR := $(call getLocalUserDir,$(LOCAL_USER_DIR))
+
+
+
 # build var: SQL_DEFINE
 # Список значений для переменных SQL*Plus.
 #
