Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 550)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -21,6 +21,7 @@
   install-after         \
   install-before        \
   install-data          \
+  install-load          \
   install-schema        \
 
 
@@ -368,6 +369,15 @@
 
 
 
+# target: install-load
+# Загружает объекты в БД при выполнении установки.
+# Список загружаемых объектов такой же, как для цели load ( берется из
+# переменной loadTarget).
+
+install-load: install-load.oms
+
+
+
 # Каталоги для загрузки установочных данных.
 # Всегда загружаются данные из каталога Install/Data/Last. Дополнительно при
 # установке обновления ( INSTALL_VERSION=N.N.N) загружаются данные из каталога
