Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 336)
+++ Config/NewModule/DB/Makefile	(revision 337)
@@ -83,7 +83,7 @@
 # group: Загрузка файлов в БД
 #
 
-# Функции для разбора строки подключения
+# Функции для разбора строки подключения.
 
 # Выделяет имя пользователя из loadUserId.
 getUserName	= $(firstword $(subst /, ,$(firstword $(subst @, ,$(loadUserId)))))
