Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 407)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -240,8 +240,8 @@
 # ƒействи€, выполн€емые перед установкой модул€.
 installBeforeTarget =													\
 	$(addsuffix .$(ru),													\
-		$(wildcard Install/Action/$(INSTALL_VERSION)/before.sql)	\
-		$(wildcard Install/Action/before.sql)			\
+		$(wildcard Install/Config/$(INSTALL_VERSION)/before-action.sql)	\
+		$(wildcard Install/Config/before-action.sql)			\
 		$(installBeforeOmsAction)									\
 	)																						\
 	$(addsuffix .$(ru2),												\
@@ -295,8 +295,8 @@
 installAfterTarget =													\
 	$(addsuffix .$(ru),													\
 		$(installAfterOmsAction)									\
-		$(wildcard Install/Action/after.sql)			\
-		$(wildcard Install/Action/$(INSTALL_VERSION)/after.sql)	\
+		$(wildcard Install/Config/after-action.sql)			\
+		$(wildcard Install/Config/$(INSTALL_VERSION)/after-action.sql)	\
 	)																						\
 	$(addsuffix .$(ru2),												\
 		$(installAfterOmsAction)									\
