Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 1714)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -623,10 +623,10 @@
 # Скрипты для загрузки пакетных заданий и их параметров в БД.
 installBatchTarget = \
   $(addsuffix .$(ru), \
-	  $(wildcard Install/Batch/Last/*.job.sql) \
-	  $(wildcard Install/Batch/Last/*/*.job.sql) \
-	  $(wildcard Install/Batch/Last/*/batch.xml) \
-  )
+    $(wildcard Install/Batch/Last/*.job.sql) \
+    $(wildcard Install/Batch/Last/*/*.job.sql) \
+    $(wildcard Install/Batch/Last/*/batch.xml) \
+  ) \
   $(addsuffix .$(ru2), \
   ) \
   $(addsuffix .$(ru3), \
