Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 746)
+++ Config/NewModule/DB/Makefile	(revision 747)
@@ -10,6 +10,7 @@
 .PHONY:                 \
   all                   \
   clean                 \
+  set-version           \
   show-oms-version      \
   update-oms-version    \
   gendoc                \
@@ -52,6 +53,39 @@
 
 
 #
+# group: Версия модуля
+#
+
+# build var: MODULE_VERSION
+# Номер текущей версии модуля.
+# Должен быть явно задан при использовании цели <set-version>.
+# Если не задан явно, то при выполнении целей, работающих с БД, автоматически
+# определяется по значению тэга version в map-файле ( Doc/map.xml). При этом
+# в случае, если в <INSTALL_VERSION> указана более старшая версия, чем в
+# map-файле, и она используется в модуле ( есть соответствующие подкаталоги в
+# ветке DB/Install или документирована в DB/Doc/version.txt), то версия
+# модуля автоматически устанавливается в значение, указанное в
+# <INSTALL_VERSION> ( при этом вносятся изменения файлы модуля, аналогично
+# выполнению цели <set-version>).
+#
+MODULE_VERSION =
+
+
+
+# target: set-version
+# Устанавливает номер текущей версии модуля.
+# Новый номер версии указывается в параметре <MODULE_VERSION>. При установке
+# номера обновляется значение тэга version в map-файле ( Doc/map.xml) и тэга
+# SubTitle в меню документации ( DB/OmsModule/NaturalDocs/Menu.txt).  Новый
+# номер будет отображаться в меню документации после ее обновления ( например,
+# командой 'make gendoc').
+#
+
+set-version: set-version.oms
+
+
+
+#
 # group: Версия OMS-файлов
 #
 
