Index: Config/NewModule/DB/Makefile
===================================================================
--- Config/NewModule/DB/Makefile	(revision 580)
+++ Config/NewModule/DB/Makefile	(working copy)
@@ -317,7 +317,7 @@
 # Из списка автоматически исключаются файлы, не загружаемые из-за значений
 # параметров LOAD_FILE_MASK и SKIP_FILE_MASK.
 # Формат списка такой же, как у loadTarget.
-checkLockTarget =                             \
+installCheckLockTarget =                      \
   $(loadTarget)                               \
 
 # Стандартные действия, выполняемые перед установкой.
