Index: Config/NewModule/DB/loaddeps.mk
===================================================================
--- Config/NewModule/DB/loaddeps.mk	(revision 245)
+++ Config/NewModule/DB/loaddeps.mk	(revision 246)
@@ -6,8 +6,8 @@
 # .$(lu2)			- загрузка под вторым пользователем
 # .$(lu3)			- загрузка под третьим пользователем
 #
-# Пример ( зависимость тела пакета pkg_TestModule от спецификации при загруке
-# по первым пользователем):
+# Пример ( зависимость тела пакета pkg_TestModule от спецификации при загрузке
+# под первым пользователем):
 # 
 # pkg_TestModule.pkb.$(lu): 		\
 # 	pkg_TestModule.pks.$(lu) 		\
