Index: Config/NewModule/DB/loaddeps.mk
===================================================================
--- Config/NewModule/DB/loaddeps.mk	(revision 413)
+++ Config/NewModule/DB/loaddeps.mk	(working copy)
@@ -2,17 +2,17 @@
 # Зависимости при загрузке файлов в БД.
 #
 # Файлы в зависимостях должны указываться с дополнительным суффиксом:
-# .$(lu)			- загрузка под первым пользователем
-# .$(lu2)			- загрузка под вторым пользователем
-# .$(lu3)			- загрузка под третьим пользователем
+# .$(lu)      - загрузка под первым пользователем
+# .$(lu2)      - загрузка под вторым пользователем
+# .$(lu3)      - загрузка под третьим пользователем
 #
 # Пример ( зависимость тела пакета pkg_TestModule от спецификации при загрузке
 # под первым пользователем):
-# 
-# pkg_TestModule.pkb.$(lu): 		\
-# 	pkg_TestModule.pks.$(lu) 		\
 #
+# pkg_TestModule.pkb.$(lu):     \
+#   pkg_TestModule.pks.$(lu)     \
+#
 
-$(packageName).pkb.$(lu): 								\
-	$(packageName).pks.$(lu)								\
-	
+$(packageName).pkb.$(lu):                 \
+  $(packageName).pks.$(lu)                \
+
