diff --git a/Module/OraMakeSystem/Data/NewModule/DB/Makefile b/Module/OraMakeSystem/Data/NewModule/DB/Makefile
index a97eaed7..1d664f80 100644
--- a/Module/OraMakeSystem/Data/NewModule/DB/Makefile
+++ b/Module/OraMakeSystem/Data/NewModule/DB/Makefile
@@ -761,8 +761,8 @@ test: test.oms
 
 # build var: UNINSTALL_RESULT_VERSION
 # Версия модуля, которая получается в результате отмены установки версии модуля.
-# Используется при сохранении в БД информации об отмене установки, необходимо
-# указывать при отмене установки обновления.
+# Используется при сохранении в БД информации об отмене установки, в некоторых
+# случаях необходимо явно указывать при отмене установки обновления.
 UNINSTALL_RESULT_VERSION  =
 
 # Список объектов для проверки отсутствия блокировок перед отменой установки.
